README.md

```diff
@@ -1,3 +1,102 @@
-TuxedoJS
-========
+# TuxedoJS
+
 ![Tux Logo](Tux-full-logo.png "Tux-Logo")
+
+
+> A Front-End Javascript Framework built on Facebook’s powerful React view layer and Flux architecture.
+
+React and Flux are two novel and exciting approaches to front-end development. TuxedoJS implements a more accessible and semantic toolkit that leverages these technologies, abstracting away complexity and extending core functionality. TuxedoJS comes from Tux - a merging of React and Flux.
+
+## Table of Contents
+
+1. [Usage](#usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+    1. [Installing Dependencies](#installing-dependencies)
+    1. [Tasks](#tasks)
+    1. [Roadmap](#roadmap)
+1. [Graceful Degradation](#graceful-degradation)
+1. [Credits and Tech Stack](#credits-and-tech-stack)
+1. [Contributing](#interested-in-contributing)
+1. [Team](#team)
+
+## Usage
+
+TuxedoJS is a modular framework that provides tools for easily rendering components, client-side routing, animating transitions, validating inputs, and managing app data.  Install it via a package manager and include the modules you need to quickly start building your app.
+
```

Comment:
I think this section needs to be built out a little more. We should talk about some of the actual pieces of of Tux and what they do. Keep it mile high but maybe some real quick example pieces like:

Tux abstracts away the complexity of Flux with powerful `Actions` syntax:

```
    var Actions = require('tux/Actions');

    var todoActions = Actions.createActionCategory({
      category: 'todos',
      source: 'todo views',
      actions: ['add', 'remove']
    });

    module.exports = todoActions;
```

Tux provides all of the glue code needed to build stores and register them with the `TuxActions` dispatcher:

```
    var ActionStores = require('tux/Stores/ActionStores');

    var todoStore = ActionStores.createStore({
      _todos: [],

      getAll: function () {
        return this._todos;
      },

      onAdd: function (todo) {
        this._todos.push({
          text: todo.text,
          id: this._todos.length
        });
        this.emitChange();
      },

      onRemove: function (todo) {
        this._todos.splice(todo.id, 1);
        this.emitChange();
      },

      register: function () {
        return {
          todos: {
            add: this.onAdd,
            remove: this.onRemove
          }
        };
      }
    });

    module.exports = todoStore;
```

Tux provides powerful opinionated `React` classes that make connecting with your stores and sharing methods with child components a synch.

```
    var React = require('tux/React');

    var Todo = React.createMutableClass({
      mutableTraits: {
        props: 'text'
      },

      propTypes: {
        todo: React.PropTypes.object.isRequired
      },

      nearestOwnerPropTypes: {
        remove: React.PropTypes.func.isRequired
      },

      handleRemove: function (e) {
        e.preventDefault();
        this.nearestOwnerProps.remove(this.props.todo);
      },

      render: function () {
        return (
          <p>{ this.props.todo.text }</p>
          <button onClick={this.handleRemove}>Delete</buttion>
        );
      }
    });

    var TodoCreateForm = React.createOwneeClass({
      nearestOwnerPropTypes: {
        add: React.PropTypes.func.isRequired
      },

      handleSubmit: function (e) {
        e.preventDefault();
        var todoTextInput = this.refs.textInput.getDOMNode();

        this.nearestOwnerProps.add({
          text: todoTextInput.value
        });

        todoTextInput.value = '';
      },

      render: function(){
        return (
          <form onSubmit={this.handleSubmit}>
            <input type="text" ref="textInput" placeholder="Add a Todo" />
            <button type="submit">Add</button>
          </form>
        );
      }
    });

    var todoStore = require('./todoStore');
    var todoActions = require('./todoActions');

    var TodoViewOwner = React.createOwnerClass({
      getInitialState: function () {
        return {
          todos: todoStore.getAll()
        };
      },

      connectOwnerToStore: function () {
        return {
          store: todoStore,
          listener: function () {
            this.setState({ todos: todoStore.getAll() });
          }
        };
      },

      registerOwnerProps: function () {
        return {
          add: todoActions.add,
          remove: todoActions.remove
        };
      },

      render: function () {
        var Todos = this.state.todos.map(function(todo) {
          return (
            <Todo todo={todo} />
          );
        });

        var 

        return (
          <div>
            <TodoCreateForm />
            {Todos}
          </div>
        );
      }
    });
```